### PR TITLE
feat(CONT-1790): Move to ssdk 16.0.1

### PR DESCRIPTION
**Pull request by kfolwarczny**

### CODE REVIEW

File: contact-manager-integration-webhook-sync-extension/pom.xml [hunk 1]
```diff
@@ -16,28 +16,21 @@
     <properties>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
-        <swagger-annotations.version>1.6.8</swagger-annotations.version>
+        <swagger-annotations.version>2.2.15</swagger-annotations.version>
```
Comment (pawelslowik) [inline contact-manager-integration-webhook-sync-extension/pom.xml]: Do we need that version here? It comes from SSDK parent pom, so I guess we could skip it here.

**Review by pawelslowik — DISMISSED** [review]

File: contact-manager-integration-webhook-sync-extension/pom.xml (hunk 1 above)
Comment (kfolwarczny) [reply contact-manager-integration-webhook-sync-extension/pom.xml]: removed

File: contact-manager-integration-webhook-sync-extension/pom.xml [hunk 2]
```diff
@@ -16,30 +16,21 @@
...
         <jackson-databind-nullable.version>0.2.6</jackson-databind-nullable.version>
-        <boat-maven-plugin.version>0.15.5</boat-maven-plugin.version>
-        <specs.path>${project.build.directory}/specs</specs.path>
-        <contact-manager-integration-webhook-sync-api.version>1.0.1</contact-manager-integration-webhook-sync-api.version>
-    </properties>
+        <boat-maven-plugin.version>0.17.21</boat-maven-plugin.version>
+        <archunit.skip>true</archunit.skip>
+
+        <specs.repo.url>https://repo.backbase.com/specs</specs.repo.url>
+        <wagon-maven-plugin.version>2.0.0</wagon-maven-plugin.version>
```
Comment (polarfish) [inline contact-manager-integration-webhook-sync-extension/pom.xml]: I suggest to use the latest wagon plugin version - 2.0.2

File: contact-manager-integration-webhook-sync-extension/pom.xml [hunk 3]
```diff
@@ -98,7 +87,7 @@
                         </goals>
                         <configuration>
```
Comment (polarfish) [inline contact-manager-integration-webhook-sync-extension/pom.xml:88]: Shall we better use `generate-spring-boot-embedded` goal for this execution?

**Review by polarfish — DISMISSED** [review]

File: contact-manager-integration-webhook-sync-extension/pom.xml (hunk 2 above)
Comment (kfolwarczny) [reply contact-manager-integration-webhook-sync-extension/pom.xml]: Done

File: contact-manager-integration-webhook-sync-extension/pom.xml (hunk 3 above)
Comment (kfolwarczny) [reply contact-manager-integration-webhook-sync-extension/pom.xml:88]: I would keep it as it is for now, to close the PR and the task. But that's good hint to check 👍🏻

**Review by pawelslowik — APPROVED** [review]

**Pull request merged**